agri-store-react/src/Pages/CartPage.js

```diff
@@ -0,0 +1,91 @@
+import { useContext, useEffect, useState } from "react";
+import axios from "axios";
+import "./CartPage.css";
+import { ProductListingPagecontext } from "../Components/Context/ContextProductListingPage";
+import { Header } from "../Components/Header/Header";
+function CartPage() {
+  const { cart, setCart, deleteQty, updateQty } = useContext(
+    ProductListingPagecontext
+  );
+  const [totalprice, setTotalPrice] = useState();
+
+  useEffect(() => {
+    setTotalPrice(
+      cart.reduce((acc, item) => acc + Number(item.qty) * Number(item.price), 0)
+    );
+  }, [cart]);
+
+  return (
+    <div>
+      <Header cart={cart} />
+      <div className="cart-price-container">
+        <table id="customers">
```

Comment:
![1](https://user-images.githubusercontent.com/31363773/167072374-8e2654cf-4759-48e8-8b3c-98e9447c5847.PNG)
Customers might not be the best id name here. Make sure you have the same styles for the table and the card-items. On my screen the delete button comes under the price section